src/components/app-configure/access/CreateDelegate.vue

```diff
@@ -0,0 +1,161 @@
+  selectedDelegateAddress.value = address
+}
+</script>
+
+<template>
+  <VOverlay>
+    <div class="center height-100 width-100">
+      <VCard popup="true" class="flex column" style="padding: 30px">
+        <p v-if="loader.show">{{ loader.message }}</p>
+        <VStack v-else gap="1rem" direction="column">
+          <VStack justify="space-between" align="center">
+            <h1 class="sub-heading-5 font-700 text-uppercase">
+              Create A Delegate
+            </h1>
+            <button
+              style="
+                cursor: pointer;
+                background-color: transparent;
+                border: none;
+              "
+              @click="emit('close')"
+            >
+              <img src="@/assets/iconography/close.svg" />
+            </button>
+          </VStack>
+          <VSeperator />
+          <VStack direction="column" gap="1.5rem">
+            <p class="body-3 grey">
+              Provide a name, assign a key and set permissions to your delegate
+              below
+            </p>
+            <VStack gap="0.5rem" direction="column">
+              <label class="body-3 grey">Name</label>
+              <VTextField v-model="delegateName" :no-message="true" />
+            </VStack>
+            <VStack gap="0.5rem" direction="column">
+              <label class="body-3 grey">Key</label>
+              <VStack gap="1rem">
+                <VDropdown
+                  :options="props.delegateKeys.map((item) => item.name)"
+                  style="flex: 1"
+                  trigger-class="text-ellipsis"
+                  @update:model-value="handleKeySelected"
+                />
+                <VButton
+                  label="Generate"
+                  variant="secondary"
+                  @click.stop="emit('generateKey')"
+                />
+              </VStack>
+              <p class="body-3 grey">
+                Provide a SECP256K1 public key, choose from the list or generate
+                one
+              </p>
+            </VStack>
+            <VStack gap="1rem" direction="column">
+              <h3>Permissions Provided to Delegate</h3>
+              <VStack gap="1rem">
+                <img src="@/assets/iconography/ok.svg" />
+                <p class="sub-heading-4 font-500">Download</p>
+              </VStack>
+              <VStack gap="1rem">
+                <img src="@/assets/iconography/ok.svg" />
+                <p class="sub-heading-4 font-500">Reshare</p>
+              </VStack>
+            </VStack>
+            <VStack gap="1rem" justify="center">
+              <VButton
+                label="Cancel"
+                variant="secondary"
+                @click.stop="emit('close')"
+              />
+              <VButton label="Submit" @click.stop="onSubmit" />
+            </VStack>
+          </VStack>
+        </VStack>
+      </VCard>
+    </div>
+  </VOverlay>
+</template>
+
+<style scoped>
+p {
+  margin: 0;
+}
```

Comment:
This should be part of our reset styles.

Reply:
https://github.com/arcana-network/developer-dashboard/pull/156/commits/9f7e26f0d802ca2d5df75784932102835e0c9bc9